### ISSUE TITLE
"todohighlight.keywords" is not working 

### ISSUE THREAD

**Issue by glitchlover** [jgclark/vscode-todo-highlight#8]
The extension is only highlighting todohighlight.keywordsPattern and using the default style. here is my configuration
```
//Todo highlight ...
    "todohighlight.isEnable": true,
    "todohighlight.isCaseSensitive": true,
    "todohighlight.keywords": [
        "DEBUG:", // without further details, this will use the defaultStyle
        "REVIEW:", // as will this
        {
            "text": "todo:",
            "before": {
                "contentText": "⚡" // can add text before or after the highlight
            },
            "color": "red",
            "border": "1px solid red",
            "borderRadius": "2px", //NOTE: using borderRadius along with `border` or you will see nothing change
            "backgroundColor": "rgba(0,0,0,.2)"
        },
        {
            "text": "note:", // this is further refined by the regex pattern below
            "color": "#ff0000",
            "backgroundColor": "yellow",
            "overviewRulerColor": "grey",
            "before": {
                "contentText": "💡💦" // can add text before or after the highlight
            },
            "regex": {
                "pattern": "(?<=^|\\\\s)note[:]?" // in this example, highlight `NOTE:` with or without the `:` and that's not part of another word.  (I.e.: The above will highlight 'NOTE' but not the "note" in 'SIDENOTE').
                /**
                    * NOTE: remember to escapse the backslash if there's any in your regexp (using \\\\ instead of single backslash)"
                    * Positive lookbehind (`(?<=...)`) is only supported in Node.js v9 and up.
                    * If your VSCode version is built on an earlier version the example above may not work.
                    * bynoted byNOTE NOTEing NOTE:
                    **/
            },
            "isWholeLine": false
        },
        {
            "text": "topic:",
            "color": "green",
            "backgroundColor": "rgba(0,0,0,0)", // INFO: setting the last parameter to zero (alpha channel) disables the background colour
            "border": "none",
            "before": {
                "contentText": "🚀" // can add text before or after the highlight
            },
            "isWholeLine": false
        },
    ],
    "todohighlight.keywordsPattern": "todo:|fixme:|pop:|note:|topic", //highlight `TODO:`,`FIXME:` or content between parentheses
    "todohighlight.defaultStyle": {
        "color": "white",
        "backgroundColor": "#9930ff",
        "cursor": "pointer",
        "borderRadius": "3px",
        "padding":"1px",
        "isWholeLine": false,
        //other styling properties goes here ... 
    },
    "todohighlight.include": [
        "**/*.js",
        "**/*.jsx",
        "**/*.ts",
        "**/*.tsx",
        "**/*.html",
        "**/*.php",
        "**/*.css",
        "**/*.scss",
        "**/*.md",
        "**/*.mmd",
        "**/*.markdown",
        "**/*.mdown",
        "**/*.txt",
        "**/*.rb",
        "**/*.go",
        "**/*.dart"
    ],
    "todohighlight.exclude": [
        "**/node_modules/**",
        "**/bower_components/**",
        "**/dist/**",
        "**/build/**",
        "**/.vscode/**",
        "**/.github/**",
        "**/_output/**",
        "**/*.min.*",
        "**/*.map",
        "**/.next/**"
    ],
    "todohighlight.maxFilesForSearch": 5120,
    "todohighlight.toggleURI": false
}
```
and this is how it looks:
![image](https://user-images.githubusercontent.com/59877631/92993484-c0707280-f513-11ea-9345-41012440e9f7.png)

**Comment by jgclark** [comment]
@glitchlover, there are two things to change to get what you want.

1. change the `\\\\` to `\\` in the regex `pattern:` line.  (The documentation I inherited seems to be wrong here. I've now corrected it.)
2. remove the `todohighlight.keywordsPattern` line. The README says about this line:

> NB: if this is present, `todohighlight.keywords` will be ignored.

This is then what I get:
![todo-highlight-test@2x](https://user-images.githubusercontent.com/1671754/93688567-9fcd8d00-fabe-11ea-8e9d-6bd2910ea5fd.jpg)
I hope you get the same.
